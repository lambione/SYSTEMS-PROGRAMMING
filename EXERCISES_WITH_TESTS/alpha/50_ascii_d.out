QcbQeaQcdQceQcfQfoQcgQckQciQcjQdaQdbQdcQddQdeQdfQdgQdhQdiQdjQcnQfpQdnQdnQhlQhnQf
lQfnQdlQdkQchQccQfmQhmQcmQdmQcoQdoQcpQdpQfmQhmQccQakQcbQeaQcdQceQcfQfoQcgQckQciQ
cjQdaQdbQdcQddQdeQdfQdgQdhQdiQdjQcnQfpQdnQdnQhlQhnQflQfnQdlQdkQchQccQfmQhmQcmQdm
QcoQdoQcpQdpQfmQhmQccQakQcbQeaQcdQceQcfQfoQcgQckQciQcjQdaQdbQdcQddQdeQdfQdgQdhQd
iQdjQcnQfpQdnQdnQhlQhnQflQfnQdlQdkQchQccQfmQhmQcmQdmQcoQdoQcpQdpQfmQhmQccQak