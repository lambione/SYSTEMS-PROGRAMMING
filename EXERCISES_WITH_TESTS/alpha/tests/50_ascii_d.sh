echo -n QcbQeaQcdQceQcfQfoQcgQckQciQcjQdaQdbQdcQddQdeQdfQdgQdhQdiQdjQcnQfpQdnQdnQhlQhnQflQfnQdlQdkQchQccQfmQhmQcmQdmQcoQdoQcpQdpQfmQhmQccQakQcbQeaQcdQceQcfQfoQcgQckQciQcjQdaQdbQdcQddQdeQdfQdgQdhQdiQdjQcnQfpQdnQdnQhlQhnQflQfnQdlQdkQchQccQfmQhmQcmQdmQcoQdoQcpQdpQfmQhmQccQakQcbQeaQcdQceQcfQfoQcgQckQciQcjQdaQdbQdcQddQdeQdfQdgQdhQdiQdjQcnQfpQdnQdnQhlQhnQflQfnQdlQdkQchQccQfmQhmQcmQdmQcoQdoQcpQdpQfmQhmQccQak | "$1" -d
