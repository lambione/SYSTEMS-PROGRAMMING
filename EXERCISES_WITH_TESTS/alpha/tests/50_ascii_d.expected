!@#$%^&*()0123456789-_=={}[];:'"\|,<.>/?\|"
!@#$%^&*()0123456789-_=={}[];:'"\|,<.>/?\|"
!@#$%^&*()0123456789-_=={}[];:'"\|,<.>/?\|"
